client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -240,7 +241,7 @@ const NotificationSubscriptions = createReactClass( {
 						>
 							{ this.state.submittingForm
 								? this.props.translate( 'Saving…' )
-								: this.props.translate( 'Save Notification Settings' ) }
+								: this.props.translate( 'Save notification settings' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.